index.js

```diff
@@ -302,28 +269,27 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
     repo = getAddedRepoConfigName(payload)
     if (repo) {
       return syncSettings(false, context, repo)
-    } 
+    }
 
     let suborg = getModifiedSubOrgConfigName(payload)
     if (suborg) {
+      robot.log.debug(`modified Suborg ${JSON.stringify(suborg)}`)
       return syncSubOrgSettings(false, context, suborg)
     }
 
     suborg = getAddedSubOrgConfigName(payload)
     if (suborg) {
       return syncSubOrgSettings(false, context, suborg)
     }
-    
+
     if (!settingsModified) {
       robot.log.debug(`No changes in '${Settings.FILE_NAME}' detected, returning...`)
```

Comment:
```suggestion
      robot.log.debug(`No changes detected, returning...`)
```

the `if (!settingsModified) {` and closing bracket should also go away. But can´t make a suggestion due to a deleted line

Reply:
I am looking at the logic to identify if more than one file has been changed and I feel it might be possible to use the same logic to call syncSettings or syncSubOrgSettings iteratively for the changed files.

Reply:
That would be even better than the change I've suggested to synch all (it would be temporary until multiple files changes are supported).